src/main/java/it/pagopa/interop/probing/scheduler/dto/PollingEserviceResponse.java

```diff
@@ -2,10 +2,12 @@
 
 import java.util.List;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
 import lombok.Data;
 
 
 @Data
+@Builder
 public class PollingEserviceResponse {
```

Comment:
Every class need interface, in this case a marker interface as: ... implements Response